analytics/location_heatmaps/run_experiment.py

```diff
@@ -122,35 +132,70 @@ def run_experiment(true_image,
       quantize: apply quantization to the vectors.
       noise_class: use specific noise, defaults to GeometricNoise.
       save_gif: saves all images as a gif.
+      count_min: use count-min sketch.
 
   Returns:
       A list of per level geo_utls.AlgResult objects.
   """
-
-  tree, tree_prefix_list = geo_utils.init_tree()
+  config = Config(dataset=dataset,
```

Comment:
I agree, I am not really using config now, as it was more informative to pass the exact parameters around. But I'd still keep it in case we want to extend the algorithm